tests/data/hook-callback.php

```diff
@@ -240,6 +248,33 @@ function return_value_typed() : int {
 
 function no_return_value_untyped( $value ) {}
 
+/**
+ * @return mixed
+ */
+function return_value_mixed() {
+    return 123;
+}
+
+function return_value_implicit_mixed( $value ) {
+    return $value;
+}
+
+/**
+ * Return type documented as a union that includes mixed.
+ *
+ * @return int|mixed
```

Comment:
I'm sorry to report there is no `*|mixed` type.
I see it all over the Internet. Please keep this project clean.

Reply:
it will be mixed, yeah. I guess this was meant as regression test for the reported bug :)

Reply:
Yeah this is a useless union type. I assume that under the hood it's treated as `mixed` by PHPStan so it should be handled the same as `mixed`. This was indeed added as a regression test for the bug reported by Pascal.